src/Silk.NET.Core/Native/NativeExtension.cs

```diff
@@ -1,8 +1,13 @@
+// This file is part of Silk.NET.
+// 
+// You may modify and distribute Silk.NET under the terms
+// of the MIT license. See the LICENSE file for details.
+
 using AdvancedDLSupport;
 
 namespace Silk.NET.Core.Native
 {
-    public abstract class NativeExtension<T> : NativeLibraryBase where T:NativeAPI
+    public abstract class NativeExtension : NativeLibraryBase
```

Comment:
Revert this please, I want extensions to have the profile to which they belong clearly marked.